src/adc.rs

```diff
@@ -37,16 +47,18 @@ impl AdcBuffer {
     }
 }
 
+impl Default for AdcBuffer {
```

Comment:
I know it's because of clippy but I am not sure whether this lint is a good one.

Reply:
fixed.